.husky/pre-push

```diff
@@ -2,3 +2,4 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 npm run test
+git config advice.ignoredHook false
```

Comment:
We realy need this line? @Mygal-Alex , @LeonidShamarin , @ArtemHolikov , @marianamartsy , please test how it work for you.